.ci/openshift-ci/build-root/Dockerfile

```diff
@@ -0,0 +1,39 @@
+# Use the centos image as the base image
+FROM quay.io/centos/centos:stream8
+
+# Install additional tooling, see https://github.com/openshift/release/blob/master/clusters/app.ci/supplemental-ci-images/ci-tools-build-root.yaml#L338
+RUN INSTALL_PKGS="git java-17-openjdk.x86_64 httpd-tools" && \
+    dnf install -y $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    dnf clean all && \
+    touch /os-build-image && \
+    git config --system user.name fabiobrz && \
+    git config --system user.email fburzigo@redhat.com
```

Comment:
one last thing ... I think that git config isn't needed

Reply:
Makes sense, thanks. Fixing!